### PR TITLE
    'C409416 A user with "Settings (Finance): View settings" permission can only view appropriate settings (Thunderjet)(TaaS)',

### PR DESCRIPTION
**Pull request by tomleexx**
[Allure report] - https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/4979/allure/
[C409416]-https://foliotest.testrail.io/index.php?/cases/view/409416

### CODE REVIEW

File: cypress/support/dictionary/devTeams.js [hunk 1]
```diff
@@ -7,4 +7,5 @@ export default {
   volaris: 'volaris',
   ideaLabsTests: 'ideaLabsTests',
   erm: 'erm',
+  bama: 'BAMA',
```
Comment (dnovikau) [inline cypress/support/dictionary/devTeams.js:10]: Please remove, the value is not used anywhere

File: cypress/e2e/settings/finance/user-finance-view-setting-only-view-appropriate-settings.cy.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
+import devTeams from '../../../support/dictionary/devTeams';
```
Comment (dnovikau) [inline cypress/e2e/settings/finance/user-finance-view-setting-only-view-appropriate-settings.cy.js:1]: Please use 1 line import for DevTeams, Permissions, TestTypes

File: cypress/e2e/settings/finance/user-finance-view-setting-only-view-appropriate-settings.cy.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+import permissions from '../../../support/dictionary/permissions';
+import { getTestEntityValue } from '../../../support/utils/stringTools';
+import TestTypes from '../../../support/dictionary/testTypes';
+import Users from '../../../support/fragments/users/users';
+import PatronGroups from '../../../support/fragments/settings/users/patronGroups';
+import ServicePoints from '../../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import UserEdit from '../../../support/fragments/users/userEdit';
+import SettingsMenu from '../../../support/fragments/settingsMenu';
+import SettingsFinance from '../../../support/fragments/settings/finance/settingsFinance';
+import expenseClasses from '../../../support/fragments/settings/finance/expenseClasses';
```
Comment (dnovikau) [inline cypress/e2e/settings/finance/user-finance-view-setting-only-view-appropriate-settings.cy.js:11]: Please use pascal case for imported modules: ExpenseClasses, FundTypes

**Review by dnovikau — CHANGES_REQUESTED** [review]

**Comment by tomleexx** [comment]
New Allure report 
https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/5043/allure/

File: cypress/e2e/settings/finance/user-finance-view-setting-only-view-appropriate-settings.cy.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+  it(
+    'C409416 A user with "Settings (Finance): View settings" permission can only view appropriate settings (Thunderjet)(TaaS)',
+    { tags: [TestTypes.criticalPath, DevTeams.thunderjet] },
+    () => {
+      SettingsFinance.verifyItemInFinancePanel();
+      SettingsFinance.verifyItemInDetailPanel();
+      SettingsFinance.checkFundType(newFundTypes);
+      SettingsFinance.clickExpenseClass();
+      SettingsFinance.verifyItemInDetailPanel();
+      SettingsFinance.checkExpenseClass(newExpenseClass);
```
Comment (IhorBohdan) [inline cypress/e2e/settings/finance/user-finance-view-setting-only-view-appropriate-settings.cy.js:60]: Last step in test case and last step here are differnt
"Expense classes" pane appears and contains list of created expense classes
"New" button on the top right is NOT active
"Actions" column is empty and does NOT contain "Edit" (pencil) and "Delete" (trash can) icons

Comment (IhorBohdan) [reply cypress/e2e/settings/finance/user-finance-view-setting-only-view-appropriate-settings.cy.js:60]: You need to add method, whitch will be check, tha user can not edit or delete expense classes

Comment (tomleexx) [reply cypress/e2e/settings/finance/user-finance-view-setting-only-view-appropriate-settings.cy.js:60]: verifyItemInDetailPanel() {
    cy.expect([
      actions.absent(),
      newButton.has({ disabled: true }),
    ]);
  },
  
  I have checked in this method, please confirm

Comment (tomleexx) [reply cypress/e2e/settings/finance/user-finance-view-setting-only-view-appropriate-settings.cy.js:60]: > You need to add method, whitch will be check, tha user can not edit or delete expense classes

beacuse I can not find the Action button,So we also can not find edit and delete icon in dropdown of action

Comment (IhorBohdan) [reply cypress/e2e/settings/finance/user-finance-view-setting-only-view-appropriate-settings.cy.js:60]: Screenshot_343

Comment (IhorBohdan) [reply cypress/e2e/settings/finance/user-finance-view-setting-only-view-appropriate-settings.cy.js:60]: Screenshot_344

Comment (tomleexx) [reply cypress/e2e/settings/finance/user-finance-view-setting-only-view-appropriate-settings.cy.js:61]: Add below function ,please check
checkEditDeleteIcon: (typeClass) => {
    cy.do(
      MultiColumnListCell({ content: typeClass.name }).perform((element) => {
        const rowNumber = element.parentElement.parentElement.getAttribute('data-row-index');
        cy.expect(getEditableListRow(rowNumber).find(editIconButton).absent());
        cy.expect(getEditableListRow(rowNumber).find(deleteButton).absent());
      }),
    );
  },

**Review by IhorBohdan — APPROVED** [review]

**Pull request merged**